orch/cattle/cattle.go

```diff
@@ -143,32 +131,10 @@ func (orc *cattleOrc) createVolume(volume *types.VolumeInfo) (*types.VolumeInfo,
 		RancherCompose: stackBuffer(rancherComposeTemplate, volume).String(),
 		StartOnCreate:  true,
 	}
-	stack, err := orc.rancher.Stack.Create(stack0)
+	_, err := orc.rancher.Stack.Create(stack0)
 	if err != nil {
 		return nil, errors.Wrapf(err, "failed to create stack '%s'", stack0.Name)
 	}
-	volume.Replicas = genReplicas(volume.NumberOfReplicas)
-
-	for _, replica := range volume.Replicas {
-		_, err := orc.rancher.Container.Create(orc.replicaContainer(volume, replica))
-		if err != nil {
-			return nil, errors.Wrapf(err, "error creating replica '%s', volume '%s'", replica.Name, volume.Name)
-		}
-	}
-
-	if err := util.Backoff(30*time.Second, "timed out", func() (bool, error) {
```

Comment:
removing the backoff here results in `Could not find service 'controller'` error thrown from `getVolume()`